### ISSUE TITLE
Error: Unknown Lexem type: ATTRIB_NAME

### ISSUE THREAD

**Issue by ASaiAnudeep** [nirgit/simple-xml-to-json#10]
I'm facing an issue while trying to parse an xml file from TestNG.

```
simple-xml-to-json/src/transpiler.js:80
                throw new Error("Unknown Lexem type: " + lexem.type)
                ^

Error: Unknown Lexem type: ATTRIB_NAME
    at parseExpr (/node_modules/simple-xml-to-json/src/transpiler.js:80:23)
```

XML File:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="20" total="40" passed="8" failed="11" skipped="1">
	<reporter-output></reporter-output>
	

</testng-results>
```

The issue occurs if there is tab space - `\t` after a new line.

**Comment by nirgit** [comment]
Thank you for reporting, 
a fix should be released soon.

**Comment by nirgit** [comment]
(should be) fixed in version 1.0.8
fyi @ASaiAnudeep
